state/account.go

```diff
@@ -23,6 +23,12 @@ var (
 	ErrAccountCollision = errors.New("account already exists")
 )
 
+// EmptyAccount indicates an empty account
+var EmptyAccount = &Account{
```

Comment:
can you add a comment that this is a read-only var for comparison
caller should not modify it

Reply:
also move it to the var() group above